scripts/setup.sh

```diff
@@ -5,6 +5,6 @@ test_mnemonic="wage thunder live sense resemble foil apple course spin horse gla
 
 persistenceCore init test --chain-id test
 echo $test_mnemonic | persistenceCore keys add test --recover --keyring-backend test
-persistenceCore add-genesis-account test 100000000000000stake --keyring-backend test
-persistenceCore gentx test 10000000stake --chain-id test --keyring-backend test
+persistenceCore add-genesis-account test 100000000000000uxprt 100000000000000stake --keyring-backend test
```

Comment:
must be `100000000000000uxprt,100000000000000stake`

```
Error: accepts 2 arg(s), received 3
Usage:
  persistenceCore add-genesis-account [address_or_key_name] [coin][,[coin]] [flags]
```

Reply:
Done.